src/documents/tutorial-prefsFramework/CreatingAPrefsEditor.md

```diff
@@ -0,0 +1,779 @@
+
+##### Preferences #####
+The next thing in the Auxiliary Schema is the configuration for the auto-pilot preference:
+
+```javascript
+autoPilot: {
+    // this 'type' must match the name of the pref in the primary schema
+    type: "minEditor.autoPilot",
+
+    panel: {
+        // this 'type' must match the name of the panel grade created for this pref
+        type: "minEditor.panels.autoPilot",
+
+        // selector indicating where, in the main template, to place this panel
+        container: ".mec-autoPilot",
+
+        // the template for this panel
+        template: "%templatePrefix/autoPilot.html"
+    }
+}
+```
+
+(The name of the property, `autoPilot`, can actually be anything, but it’s helpful to use the name
+of the preference.)
+
+In our example, the auto-pilot preference configuration includes two things:
+1. the type of the preference, and
+2. information about the Panel.
+
+The value of the `type` property is the name of the preference as defined in the Primary Schema.
+
+The value of the `panel` property is a JavaScript object containing configuration information
+for the Panel. Let’s look at each of the properties:
+<dl>
+<dt>`type`</dt>
+<dd>This is the name of the Panel that was defined in the call to `fluid.defaults()` above.</dd>
+<dt>`container`</dt>
+<dd>This is a CSS-based selector referencing the Panel’s placeholder element in the main HTML
+template – the one referenced by the `template` property above.</dd>
+<dt>`template`</dt>
+<dd>This is the path and filename of the HTML template for this Panel.
+Note, in this example, how the `templatePrefix` term is being used.</dd>
+</dl>
+
+#### Instantiation ####
+The last thing in the `js/minEditor.js` file is a call to the Preferences Framework
+function [`fluid.prefs.create()`](../PreferencesEditor.md). This function actually creates the Preference Editor.
+It accepts two arguments:
+1. a CSS selector indicating the container element for the Preference Editor, and
+2. a JavaScript object containing configuration information for the Preference Editor.
+
+```javascript
+minEditor.init = function (container) {
+    return fluid.prefs.create(container, {
+        build: {
+            gradeNames: ["minEditor.auxSchema"]
+        }
+    });
+};
+```
+This function is invoked in the main HTML file for the Preference Editor, `index.html`.
+You can see the entire file here:
+https://github.com/fluid-project/infusion/tree/master/examples/framework/preferences/minimalEditor/index.html.
+Let’s look at this invocation:
+
+```
+<div id="myMinEditor"></div>
+
+<script type="text/javascript">
+    minEditor.init("#myMinEditor");
+</script>
+```
+
+In the HTML snippet above, the `<div>` is the container that the Preference Editor will be
+rendered inside of. The call to `minEditor.init()` is passed the ID of the element,
+`“#myMinEditor”`, as the container argument.
+
+In the code snippet above, the first argument – `container` – is the CSS identifier passed in to
```

Comment:
Typo "in to" should be "into" assuming I'm understanding this correctly http://data.grammarbook.com/blog/definitions/into-vs-in-to-expanded/